### DISCUSSION TITLE
Pod install fails on Codemagic but works locally

### DISCUSSION THREAD

**Discussion by rasikhq** [codemagic-ci-cd/codemagic-docs#2440] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

- Try to build an Expo managed SDK 50 React Native project

I would like to point out that the Android build is working just as expected. I haven't changed any configurations, I have also tried explicitly setting XCode & Cocoapods version to match my local setup and the build still fails.

The last iOS build id that passed was for iOS was: 65fd5a2f82a4503a08f4d00f

Any help would be much appreciated

### Expected results

- `npx --yes expo prebuild` should pass including pod install

### Actual results

```
[!] The following Swift pods cannot yet be integrated as static libraries:

The Swift pod `GTMAppAuth` depends upon `GTMSessionFetcher` and `AppAuth`, which do not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.
Couldn't install Pods. Updating the Pods project and trying again...
- Couldn't install Pods. Updating the Pods project and trying again...
⚠️  Something went wrong running `pod install` in the `ios` directory.
Command `pod install` failed.
└─ Cause: The following Swift pods cannot yet be integrated as static libraries:

The Swift pod `GTMAppAuth` depends upon `GTMSessionFetcher` and `AppAuth`, which do not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.```

### Build id (optional)

6603fd455c804f2519e1b47b

**Comment by santigp258** [comment, marked as answer]
Hey, I was getting the same error using EAS. I found the solution in the following thread:
https://github.com/react-native-google-signin/google-signin/issues/1263

The error is caused by the [@react-native-google-signin/google-signin@11.0.0](https://github.com/react-native-google-signin/google-signin) package. 
What I did, was update the package to version [^11.0.1](https://github.com/react-native-google-signin/google-signin/commit/243ab99b3d12cf7e26e4e2c22012db25e148b40d) and my build ran fine.

**Reply by mateuscesarglima** [reply]
Thanks, helped me a lot!
